.github/workflows/node-ci.yml

```diff
@@ -278,11 +278,8 @@ jobs:
 
     publish_snapshot:
         name: Publish npm snapshot
-        # We don't publish snapshots on draft PRs or
-        # on the main Changeset "Version Packages" PR
-        if: |
-            github.event.pull_request.draft == false
-            && !startsWith(github.head_ref, 'changeset-release/')
+        # We don't publish snapshots on Changeset "Version Packages" PRs
+        if: !startsWith(github.head_ref, 'changeset-release/')
```

Comment:
The snapshot build didn't get created for this PR when it was in draft state... I wonder if that's because the workflow config on `main` is always used? (that seems reasonable... otherwise it'd be a security risk for OSS projects)

Reply:
I can't find the documentation for this, but I'm pretty sure you're correct. GitHub Actions use the base branch's configuration so you have to land the change before you can test/see it.